contracts/account/extensions/AccountSignerERC7702.sol

```diff
@@ -3,12 +3,12 @@
 pragma solidity ^0.8.20;
 
 import {ECDSA} from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import {AccountCore} from "../AccountCore.sol";
+import {AbstractSigner} from "../../utils/cryptography/AbstractSigner.sol";
 
 /**
  * @dev {Account} implementation whose low-level signature validation is done by an EOA.
  */
-abstract contract AccountSignerERC7702 is AccountCore {
```

Comment:
This is no longer an extension of account?

Reply:
Yeah, I would agree with this change. The reasoning is that with ERC-7702 some users may want to rely on AccountSignerERC7702 to use with a separate version of ERC-7821 (see [this PR](https://github.com/OpenZeppelin/openzeppelin-community-contracts/pull/61))

Reply:
Moved